.vscodeignore

```diff
@@ -1,7 +1,6 @@
 .vscode/**
 out/**
 src/**
-preview-src/**
```

Comment:
Does `webviews` need to be added here now? We need to stay under 25 files to publish for web.